packages/material/scss/spreadsheet/_layout.scss

```diff
@@ -51,40 +30,3 @@
     }
 
 }
-
-
-
-
-@include exports( "spreadsheet/window/layout/material" ) {
-
-    .k-spreadsheet-window {
```

Comment:
The `.k-spreadsheet-window` styles could use a further positioning tweak (could be releated to rendering also):

![image](https://user-images.githubusercontent.com/16972160/123603420-89e5b900-d802-11eb-85d9-0963996a3957.png)